src/tests/ftest/object/array_obj_test.py

```diff
@@ -26,54 +26,24 @@
 import os
 import time
 import traceback
-import sys
-import json
 import logging
-from avocado import Test, main
+from avocado import main
```

Comment:
Same as above comment, is this used?

Reply:
Same as above, yes:
https://github.com/daos-stack/daos/blob/11af10998e2bb694b92e6aeb7d31ec80ea10a8aa/src/tests/ftest/object/array_obj_test.py#L137